python/paddle/fluid/dygraph/dygraph_to_static/utils.py

```diff
@@ -181,3 +181,39 @@ def update_args_of_func(node, dygraph_node, method_name):
 
     node.args = []
     node.keywords = added_keywords + node.keywords
+
+
+def insert_gast_stmt_before_node(cur_node_wrapper, pre_nodes=[], post_nodes=[]):
```

Comment:
This class is useless now, I will delete it in a new PR or after reviewer comment so that I don't have to re-run the CI :-)